mdm/common/src/main/java/com/gentics/mesh/dagger/MeshBuilderFactory.java

```diff
@@ -0,0 +1,14 @@
+package com.gentics.mesh.dagger;
```

Comment:
This is a duplicate of MeshBuilderFactory in core, should this have been moved here?